models/netsuite2/netsuite2__balance_sheet.sql

```diff
@@ -50,13 +64,17 @@ balance_sheet as (
             and {{ dbt.date_trunc('year', 'reporting_accounting_periods.starting_at') }} = {{ dbt.date_trunc('year', 'transaction_accounting_periods.starting_at') }} 
             and reporting_accounting_periods.fiscal_calendar_id = transaction_accounting_periods.fiscal_calendar_id) then 'Net Income'
       when not accounts.is_balancesheet then 'Retained Earnings'
+      when lower(accounts.special_account_type_id) = 'retearnings' then 'Retained Earnings'
+      when lower(accounts.special_account_type_id) IN ('cta-e', 'cumultransadj') then 'Cumulative Translation Adjustment'
```

Comment:
All of these retained earnings and CTA updates don't seem to be called out in the CHANGELOG. Can we add the reasons for these additions.

Reply:
Just a small format consistency request.
```suggestion
      when lower(accounts.special_account_type_id) = 'retearnings' then 'Retained Earnings'
      when lower(accounts.special_account_type_id) in ('cta-e', 'cumultransadj') then 'Cumulative Translation Adjustment'
```

Can this be applied to the other instances.

Reply:
Updated IN to in and updated changelog.